internal/web/server.go

```diff
@@ -468,6 +498,49 @@ func defaultKubeconfigExists() bool {
 	}
 }
 
+func (s *server) initInformer() (cache.Store, cache.Controller) {
+	return cache.NewInformer(
+		&cache.ListWatch{
+			ListFunc: func(options metav1.ListOptions) (runtime.Object, error) {
+				var pkgList v1alpha1.PackageList
+				err := s.pkgClient.Packages().GetAll(context.Background(), &pkgList)
```

Comment:
I think it would be good to use the `cmd.Context()` here, which was removed from the `Start` function last week :upside_down_face: